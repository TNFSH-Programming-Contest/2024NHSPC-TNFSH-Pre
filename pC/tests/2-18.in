81 1522
34 11
41 38
23 54
14 16
78 75
54 63
60 12
13 41
57 28
80 33
72 52
48 2
10 3
13 15
17 72
20 41
45 18
41 76
65 16
69 59
75 57
9 31
3 32
43 48
57 47
71 56
44 21
1 38
15 21
35 59
8 6
77 55
39 38
78 80
9 19
69 41
81 8
7 59
24 28
56 60
54 62
49 7
49 17
22 3
2 75
29 38
34 59
66 55
46 3
71 4
34 22
79 61
9 33
65 77
27 23
44 70
50 67
49 26
61 11
74 19
5 45
46 50
60 17
57 54
20 34
59 60
50 19
74 38
63 80
26 34
76 15
9 51
40 65
6 22
36 78
28 4
11 65
47 6
2 4
2 65
40 75
5 51
33 30
64 69
48 14
63 11
4 32
49 14
54 81
1 58
10 1
29 48
16 56
61 2
47 22
80 17
9 59
69 40
13 48
59 4
39 61
78 74
72 59
57 30
22 46
26 72
53 37
79 9
13 49
27 70
30 77
30 38
3 53
9 40
39 34
66 49
79 59
26 74
39 58
35 68
5 60
14 13
36 42
19 26
28 8
67 55
41 37
54 31
48 12
34 71
29 3
16 69
33 28
6 53
18 64
66 18
43 68
12 25
48 57
6 71
54 61
20 81
74 10
61 21
45 32
34 65
1 35
70 3
33 78
62 59
22 35
46 44
76 5
61 25
19 16
1 71
14 6
61 34
11 35
33 68
77 53
5 30
55 15
36 21
70 78
72 53
81 46
26 50
63 1
74 69
28 30
74 28
23 78
70 13
7 62
63 70
5 35
59 81
64 77
24 67
12 46
8 34
31 66
14 62
15 69
9 66
26 5
1 8
20 21
31 56
78 61
41 61
42 11
55 11
79 3
66 75
49 59
14 1
64 7
65 45
30 32
60 50
48 28
55 37
66 74
16 60
38 49
37 4
1 25
66 64
79 37
79 46
30 67
6 33
71 9
9 70
72 68
29 35
72 1
41 63
38 7
52 61
12 11
7 41
73 75
56 53
59 2
79 6
27 11
70 49
59 3
48 69
52 3
50 73
67 76
15 3
50 42
77 47
44 19
58 5
19 49
51 24
4 78
11 4
49 65
57 36
73 56
68 19
31 42
65 38
61 75
63 40
37 50
10 24
55 64
29 71
73 34
68 12
58 60
48 16
38 61
52 73
76 53
19 3
12 23
21 70
43 50
68 2
57 53
43 46
10 6
33 41
76 50
4 27
67 6
60 75
73 15
75 33
71 28
53 2
67 27
66 51
25 39
53 35
47 69
37 30
58 57
23 74
35 55
17 42
69 30
38 72
73 29
43 30
34 9
24 75
53 7
75 11
73 40
12 16
3 9
4 57
21 19
8 46
8 39
80 8
55 19
17 67
22 2
38 13
46 74
25 37
14 57
66 7
29 52
43 57
29 68
67 64
32 75
45 68
32 70
34 15
38 14
7 13
23 59
75 59
31 11
66 69
12 55
46 65
5 10
62 57
58 72
66 45
68 15
38 50
10 12
5 47
44 29
47 75
51 32
42 64
40 8
69 81
18 53
8 20
73 27
10 39
22 67
72 43
58 7
50 36
54 56
1 66
17 63
10 26
74 14
24 80
56 25
56 23
58 10
45 42
53 67
32 76
38 21
62 41
77 15
23 33
4 36
33 15
54 75
33 7
23 4
22 43
78 39
48 56
49 58
80 48
17 41
45 81
35 24
61 3
14 30
70 6
25 75
43 71
4 58
44 24
20 2
80 37
5 72
59 67
50 5
44 72
35 17
70 66
2 21
8 21
32 1
38 56
21 3
62 63
30 8
30 10
66 15
44 48
33 37
81 33
3 47
70 62
62 36
70 37
19 37
3 24
34 4
38 51
60 31
23 55
30 73
74 48
38 54
23 77
1 15
54 34
5 7
50 62
29 22
60 39
27 28
37 26
57 80
23 65
73 59
54 65
5 25
78 62
56 7
37 29
78 11
4 62
3 5
76 52
34 42
71 19
39 17
26 51
15 10
79 25
19 59
60 42
25 64
12 15
62 44
72 15
18 65
65 73
12 77
2 73
80 56
19 42
15 7
55 20
14 26
81 42
38 62
11 17
26 36
59 63
10 7
1 7
2 80
48 67
25 15
2 51
33 58
79 43
2 71
68 37
53 63
13 29
13 31
81 48
51 40
27 38
28 69
3 57
26 76
80 36
47 15
69 4
78 14
47 70
10 31
32 33
45 76
56 2
61 27
5 42
53 46
70 56
32 65
34 19
48 15
39 36
56 35
73 42
36 44
26 64
41 71
24 41
56 17
65 28
38 28
56 49
35 40
34 43
38 35
63 29
26 80
35 60
74 39
46 66
13 81
73 57
80 72
74 47
8 37
64 30
12 72
50 79
7 52
72 7
53 59
42 18
31 34
64 41
23 2
36 68
34 35
54 24
37 1
2 74
8 71
1 20
38 4
20 24
10 50
16 74
2 6
56 1
28 15
4 53
5 52
5 21
47 68
64 70
13 63
36 49
2 10
63 23
49 73
40 38
49 71
53 36
10 36
20 71
10 78
6 15
67 13
31 68
67 32
17 2
30 12
55 52
35 48
35 80
59 57
66 35
29 62
48 25
2 77
46 75
35 79
54 64
61 30
66 40
42 61
14 5
23 10
53 70
63 55
45 46
15 78
24 74
20 66
18 52
3 17
37 57
48 53
78 29
19 62
27 62
65 41
27 59
38 31
51 71
44 12
32 27
41 25
2 49
51 45
64 13
55 58
58 56
12 8
6 27
46 39
33 43
20 43
1 2
53 79
35 39
70 69
68 52
37 62
76 44
81 10
15 16
66 42
73 80
71 76
4 65
9 42
42 38
16 70
14 55
25 81
80 18
80 59
62 80
16 76
20 56
29 16
41 67
62 8
41 47
13 16
40 13
73 48
52 57
9 81
3 75
25 72
26 16
62 67
81 43
80 58
11 50
54 80
15 49
58 12
64 10
50 57
7 45
2 27
3 49
27 57
3 27
55 68
1 69
48 11
9 53
8 58
4 48
35 36
11 33
80 14
35 27
23 13
70 39
65 33
59 54
66 61
51 31
44 37
39 23
62 3
79 36
34 79
51 57
6 20
42 35
9 60
2 50
25 11
53 38
46 20
39 71
36 38
33 66
48 19
2 57
27 63
60 66
65 24
57 81
44 13
79 62
2 33
67 51
27 69
80 77
18 41
81 71
40 54
26 79
41 80
2 11
17 19
69 55
27 52
34 13
19 35
70 15
9 64
6 56
18 72
68 24
19 24
4 68
75 14
57 21
3 42
16 35
63 43
48 77
69 75
6 3
11 21
2 55
28 3
60 15
32 36
67 5
6 37
25 32
40 48
20 49
69 37
64 32
68 46
65 67
48 32
36 5
14 76
71 31
28 39
75 12
22 4
9 43
7 26
78 79
19 65
39 29
68 69
76 30
39 4
22 27
45 35
74 17
58 64
20 73
35 57
2 54
77 33
24 18
36 22
20 50
54 76
16 44
54 73
50 69
65 37
28 23
16 27
42 80
76 38
7 71
56 68
34 72
15 54
25 47
8 65
78 22
65 72
5 66
27 18
67 57
61 32
62 10
14 29
42 29
15 51
21 25
39 13
39 67
77 25
4 12
50 49
38 22
23 79
69 78
20 14
10 76
23 58
8 68
55 44
28 2
75 20
46 60
74 37
73 77
13 33
37 47
23 19
80 68
58 76
49 18
10 9
25 30
62 32
80 27
64 27
24 4
47 63
2 69
58 81
40 23
55 27
53 22
37 10
76 75
3 54
61 57
4 1
30 80
2 5
70 29
77 8
31 62
15 37
18 2
37 60
34 18
53 47
7 18
7 77
5 62
74 64
43 60
71 55
38 47
22 20
54 6
12 50
33 20
60 72
56 39
33 14
31 70
35 25
72 24
49 48
1 33
43 18
56 65
34 49
57 64
17 31
73 4
15 23
72 22
52 51
79 2
17 8
74 59
29 47
26 40
78 38
79 69
23 11
28 43
30 16
76 21
30 42
74 81
54 1
20 4
21 29
27 71
78 9
70 80
39 31
55 53
66 65
1 26
33 53
74 21
39 75
19 2
26 21
65 50
61 36
34 17
15 50
78 12
36 34
18 46
2 45
68 10
57 8
31 33
14 17
34 58
59 55
37 59
11 74
79 22
29 75
59 66
68 28
40 5
32 50
47 81
8 31
76 57
30 68
54 25
4 61
50 72
31 52
73 17
50 25
40 25
12 31
4 10
23 8
32 37
71 52
79 64
25 73
11 37
20 45
23 67
44 65
10 47
2 66
30 74
22 40
35 14
45 40
22 16
78 17
61 29
1 46
21 62
67 14
54 49
25 69
80 29
70 46
74 36
15 63
22 30
10 70
39 15
42 6
42 15
66 72
79 70
46 47
76 9
45 62
9 68
5 48
78 16
7 60
66 24
51 46
3 36
44 31
34 56
34 7
40 72
12 21
3 76
80 19
24 14
34 74
49 67
42 51
42 46
13 62
66 39
64 2
39 32
67 9
69 54
65 35
39 12
60 51
66 76
33 49
25 68
12 67
65 68
16 28
25 6
58 21
39 81
59 11
54 39
81 34
71 35
55 39
47 8
1 5
52 9
55 31
57 19
73 70
53 74
28 19
77 41
4 5
81 11
32 19
20 47
43 13
78 58
22 10
12 22
46 24
45 33
78 32
18 56
14 12
71 54
34 40
13 11
23 72
63 6
32 24
3 71
80 44
51 12
65 71
48 10
4 47
78 7
3 35
9 17
5 55
73 22
78 34
28 7
65 69
36 41
8 59
57 13
69 57
70 8
4 21
72 14
46 27
68 51
38 2
1 27
70 60
49 12
16 54
46 58
8 9
41 31
41 2
13 36
71 67
21 42
24 33
52 2
81 65
54 32
78 26
23 49
54 18
81 41
72 13
55 61
74 75
18 30
32 2
51 73
55 38
1 6
6 59
24 37
60 79
30 41
58 50
40 76
47 54
81 80
2 9
21 67
22 63
70 36
56 24
54 66
17 28
32 71
22 76
36 9
48 9
77 78
28 80
21 46
29 45
61 58
12 43
5 53
72 27
4 43
46 16
39 6
9 62
11 28
31 49
51 49
80 64
33 62
61 35
70 75
55 76
44 81
7 9
47 32
47 23
27 56
44 61
69 21
63 2
13 50
30 58
14 32
29 31
76 73
60 14
66 62
18 1
81 40
54 67
29 34
46 38
77 67
8 26
36 8
8 53
5 44
21 31
24 40
22 21
31 50
65 80
46 23
5 77
34 68
3 69
46 77
77 56
48 59
9 25
26 35
17 26
19 11
32 81
17 44
70 23
44 63
46 69
74 35
59 22
32 15
14 23
76 62
6 58
29 15
49 37
6 21
12 80
61 47
33 4
17 7
78 31
29 9
39 41
14 19
25 24
38 75
48 70
30 46
21 54
73 26
23 53
70 43
27 39
78 52
44 14
67 63
20 57
20 51
81 60
62 68
54 70
16 3
36 40
61 6
36 59
45 43
9 47
74 63
47 36
60 13
5 33
48 34
79 44
69 73
6 18
22 37
28 37
45 37
56 37
74 57
76 37
19 7
49 27
75 41
6 36
3 41
45 77
42 77
15 71
62 65
26 11
59 28
62 16
40 70
31 6
19 1
79 28
1 47
77 36
38 18
15 81
65 3
6 11
42 69
31 48
3 74
8 25
53 15
33 18
14 54
14 36
76 33
78 30
78 24
1 3
23 76
63 36
27 31
79 13
76 61
3 2
51 17
71 22
62 15
44 53
52 21
63 12
16 4
23 6
39 77
77 35
55 40
34 57
43 78
16 59
66 38
65 51
25 16
10 67
68 6
6 62
75 21
39 49
66 73
59 51
54 45
76 29
13 35
53 60
32 59
46 76
2 34
68 67
47 55
8 79
26 47
71 33
67 45
28 78
39 63
44 40
1 34
74 49
74 32
43 75
55 41
28 61
1 43
44 43
18 32
65 14
8 35
73 33
3 4
41 43
81 66
24 43
9 56
30 45
41 5
2 76
63 65
5 75
62 47
30 49
80 69
74 5
81 63
12 6
49 62
3 50
34 37
16 52
8 67
77 71
9 21
57 55
30 36
24 9
19 39
29 33
55 4
17 81
16 72
69 49
68 77
41 8
80 22
38 9
51 34
79 17
78 46
1 48
6 41
73 13
59 24
48 17
46 40
5 39
44 57
20 40
24 23
67 43
2 13
12 33
21 37
8 32
47 33
77 38
40 59
70 34
11 58
37 81
22 14
75 10
39 1
1 76
77 34
81 70
78 68
45 70
50 77
52 50
25 63
28 44
19 20
22 19
21 7
39 7
33 8
62 61
9 58
34 50
4 31
55 17
18 11
3 12
52 37
48 63
59 50
69 62
52 60
71 63
81 3
44 58
22 24
33 67
15 80
41 11
8 64
74 1
48 41
40 56
23 1
8 60
24 53
23 25
80 47
33 74
1 52
4 54
29 67
50 51
73 1
1 57
12 69
46 6
33 40
21 35
25 58
78 63
38 64
35 15
21 34
64 14
25 3
42 28
20 32
34 32
66 50
73 18
62 24
33 70
72 49
36 20
73 14
31 24
38 32
18 71
56 81
79 7
51 74
61 71
36 73
66 48
20 12
76 27
33 10
22 28
43 26
37 67
34 3
25 43
55 18
44 25
38 67
68 81
80 46
72 21
56 13
66 58
25 59
15 45
76 13
42 10
28 34
72 56
17 66
75 34
70 14
32 7
27 53
74 43
20 7
10 71
69 45
44 50
66 43
1 59
76 7
64 6
70 74
48 68
77 6
47 76
80 66
13 32
10 77
12 81
56 62
