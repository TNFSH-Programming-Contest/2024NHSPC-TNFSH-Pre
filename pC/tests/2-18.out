43
5 7 10 11 13 16 17 18 19 20 22 23 24 26 27 28 29 30 31 40 41 42 43 44 45 47 51 52 53 55 56 58 60 61 63 64 68 71 72 73 75 77 79 